news/docstrings-refactor.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* No news added: this branch was used to refactor the docstrings in the codebase without adding new features
```

Comment:
This warrants a news I think.  Refactored docstrings means better documentation....so someone who has an idea that the docs weren't good will find out in the CHANGELOG that they have been worked on.

Reply:
done